File: lecture_3/ascending_sequence.py
max = 0
flag = True # Trong python, ko cần phải bool, cứ thế mà gán thôi
while(True):
    x = int(input())
    if(x == 0):
        break
    elif(x > max):
        max = x 
    elif(x < max):
        flag = False
if(flag == True):
    print("YES")
elif(flag == False):
    print("NO")

File: lecture_3/ascending_sequence.cpp
#include<bits/stdc++.h>
using namespace std;
int main(){
    int x;
    int max = 0;
    bool flag = true;
    while(true){
    cin >> x;
    if(x == 0){
        break;
    }
    else if(x > max){
        max = x;
    }
    else if(x < max ){
        flag = false;
    }
    }
    if( flag == true){// TRONG BIỂU THỨC IF LUÔN DÙNG 2 DẤU BẰNG, NẾU DÙNG 1 DẤU = SẼ HIỂU LÀ GÁN 
        cout<<"YES";
    }else if(flag == false){
        cout<<"NO";
    }
    return 0;
}